packages/block-editor/src/components/block-navigation/expander.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { chevronRight, Icon } from '@wordpress/icons';
+export default function BlockNavigationExpander( { onClick } ) {
```

Comment:
Ideally in the future we don't need a component like this, and can instead use CSS rules to point at an SVG image.